posts/2019-10-15-Rustup-1.20.0.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Announcing Rustup 1.20.0"
+author: The Rustup Working Group
+---
+
+The rustup working group is happy to announce the release of rustup version 1.20.0. [Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.20.0 is as easy as:
+
+```
+rustup self update
```

Comment:
Perhaps worth menitioning that for most users, you'll get the update automatically the next time you run `rustup update`.

Reply:
Yeah we probably should, are you okay to sort a bit of wording @pietroalbini ?  Something like:

Also, rustup will automatically fetch updates for itself at the end of a normal toolchain update:

```
rustup update
```

?